### ISSUE TITLE
Forgetting to set ArticleHeadlinePhotoCredit alt text prop makes deploy fail

### ISSUE THREAD

**Issue by chrismejia** [SLB-Pizza/radio-pizza#164]
## What's the bug?

From a recent set of failed deploys: the build fails to deploy when alt text is not supplied to the 

## What's the expected behavior?

Should render an empty string if no alt text is supplied.

## Steps to Reproduce

1. Publish an editorial and forget the `article_headline_img` alt text.

## Pages & Viewports Affected

**Pages:** N/A
**Viewports:** N/A

## File(s) involved, include line(s) if known

[From the most recent failed deploy](https://app.netlify.com/teams/halfmoonbk/builds/63d9cbdadc3bf8000960b6bf)

```
9:19:45 PM: failed Building static HTML for pages - 0.825s
9:19:45 PM: error Building static HTML failed for path "/editorial/the-mighty-supr-sprt"
9:19:45 PM: 
9:19:45 PM:   66 |                 )}
9:19:45 PM:   67 |                 <ArticleHeadlinePhotoCredit
9:19:45 PM: > 68 |                   alt={article_headline_img.alt}
9:19:45 PM:      |                                             ^
9:19:45 PM:   69 |                   copyright={article_headline_img.copyright}
9:19:45 PM:   70 |                 />
9:19:45 PM:   71 |               </div>
9:19:45 PM: 
9:19:45 PM:   WebpackError: TypeError: Cannot read property 'alt' of null
9:19:45 PM:   
9:19:45 PM:   - ArticleHeadline.js:68 
9:19:45 PM:     src/components/slices/ArticleHeadline.js:68:45
```
